chassis/core/src/main/java/com/griddynamics/jagger/JaggerLauncher.java

```diff
@@ -147,7 +143,7 @@ public void run() {
         System.exit(result);
     }
 
-    private static void launchMaster(final URL directory) {
+    private static void launchMaster(final URL directory, final boolean migrateDb) {
```

Comment:
It will be much better to externalize the step. not just a  bool on start. I'd prefer to check compatibility and throw exception with message "Need to migrate DB. To do this, please execute ...